### DISCUSSION TITLE
26th release scope

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#288] [General]
As agreed during the kick-off, this is the scope for the 26th July release. 

Improvements to the current display
- Modify platform such that widgets and datasets are arranged by category
- Remove Highlighted Places widget
- Refine default scene so that mangrove extent is more visible

Improvements to functionality
- Allow users to download selected spatial data (which we own)

New data widgets
-  Climate mitigation & policy.
-- Pending on data reception. Kate should send the data by the 2nd July. 

***Only if time allows:***
- Improve place-based summary downloadable as a pdf

**Comment by dhakelila** [comment]
[Original basecamp message](https://basecamp.com/1756858/projects/16449126/messages/96909202)

**Comment by lyubovpencheva** [comment]
This is an old one.
